### PR TITLE
Add an option to stop and meditate to the tunnel storyline

### PR DESCRIPTION
**Pull request by AntonyKwok**
@casey-collab

### CODE REVIEW

File: english/tunnel/tunnel.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ hear your pursuer, only your breath echoing around you.
 [Look for a light.] (blind-search/blind-search.md)
 
 [Continue down the tunnel.] (continue/continue.md)
+
+[Stop and meditate on the situation](stop/stop.md)
```
Comment (casey-collab) [inline english/tunnel/tunnel.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by AntonyKwok** [comment]
@casey-collab

**Comment by casey-collab** [comment]
It looks like the file `english/tunnel/stop/stop.md` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by AntonyKwok** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**